scripts/security/phase4_enforcement_probe.py

```
@@ -0,0 +1,577 @@
+        for file_path in p.rglob("*"):
+            if file_path.is_file():
+                yield file_path
+
+
+def _scan_files_for_secret(scan_dirs: Iterable[str], secret: str) -> list[str]:
+    hits: list[str] = []
+    for file_path in _iter_scan_files(scan_dirs):
+        try:
+            data = file_path.read_text(encoding="utf-8", errors="ignore")
+        except Exception:
+            continue
```

Ruff (0.15.0)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)